composer.json

```diff
@@ -34,7 +34,10 @@
         }
     },
     "config": {
-        "sort-packages": true
+        "sort-packages": true,
+        "platform": {
+            "php": "7.4"
```

Comment:
I believe this needs to have the patch version too, unfortunately. I'm not at my laptop to test right now, but I thought that gave me an error. Does it work for you?

Reply:
Yes, it works, CI also run composer install for verify in php 7.4, 8.0, and 8.1

Reply:
I think mwop updated the CI so that it removes this key before running though, that's why I wanted to check.

Reply:
@internalsystemerror ok, just let me know if I can do something, thank you.